README.md

```diff
@@ -43,3 +43,62 @@ then iterates over that vector calling a method:
 The runner is fairly simplistic, but runs each benchmark in its own process
 (with 30 process executions and 100 in-process iterations), printing means and
 99% confidence intervals (calculated from the standard deviation).
+
+
+# Example results
+
+`vtable_bench` takes (optionally) 3 arguments:
+
+```
+$ cargo run --release --bin vtable_bench -- -h
+Usage: vtable_bench [-h] [<#reps> <#iters> <#vec size>]
+```
+
+From right to left (since it's easier to understand that way): `vec size` is how
```

Comment:
A bit of a nitpick perhaps, but I found this a little jarring:

 > `vec size` is how many elements the vector that's iterated will contain

How about:

>  `vec size` determines the size of the vectors.

(You've already said you will iterate them earlier)

Reply:
"Determine" is a vague word IMHO: isn't it clearer to say explicitly that it's how many elements the vector will contain?

Reply:
That's fine, but I found the original wording awkward.

So you could say, for example:

> The iterated vectors will contain `vec_size` elements.

Reply:
I'm not sure I've made it better, but see if you prefer the new phrasing.